test/BuildingRegistry.Tests/BackOffice/Api/WhenCorrectingBuildingUnitPosition/GivenBuildingHasInvalidStatusException.cs

```diff
@@ -51,7 +51,7 @@ public void ThenThrowValidationException()
                 .Result
                 .Where(x => x.Errors.Any(e =>
                     e.ErrorCode == "GebouwStatusNietInGeplandOfGerealiseerd"
-                    && e.ErrorMessage == "Deze actie is enkel toegestaan binnen een gepland of gerealiseerd gebouw."));
+                    && e.ErrorMessage == "Deze actie is enkel toegestaan binnen een gepland of gerealiseerd gebouw."));
```

Comment:
indent

Reply:
In my editor the indentation seems correct? 
![image](https://user-images.githubusercontent.com/18366653/200278708-8997b196-1238-4697-b558-a7437416ccd5.png)

Reply:
the && should be indented one level more

Reply:
Done